test/Semantics/common.sh

```diff
@@ -11,18 +11,12 @@ function die {
   exit 1
 }
 
-case $# in
-  (1) ;;
-  (2) F18=$2 ;;
-  (*) echo "Usage: $(basename $0) <fortran-source> [<f18-executable>]"; exit 1
```

Comment:
Seems like you could preserve this usage message also with a similar bit of code to that in test_folding.sh?